tests/module_tests/bsize_test.py

```diff
@@ -0,0 +1,30 @@
+import six
+import unittest
+from bsize import Size
+
+class SizeTestCase(unittest.TestCase):
+
+    def test_size(self):
+        # check failure on incorrect string
+        with six.assertRaisesRegex(self, ValueError, "does not contain size"):
+            Size("hugala buga lugala")
+
+        # check failure on malformed units
+        with six.assertRaisesRegex(self, ValueError, "Unable to identify unit"):
+            Size("1 GidB")
+
+        # accept int parameter, without units
+        self.assertEqual(Size(0).get(), "0.0 B")
+
+        # accept parameter with an exponent
+        self.assertEqual(Size("1.048576e+06B").get(), "1.0 MiB")
+
+        # accept units case insensitive, without space, convert
+        self.assertEqual(Size("1000kilObytes").get("autodec", "%d"), "1")
```

Comment:
Shouldn't there be a unit on the return value?

Reply:
Not in this case. The second parameter of `Size.get` specifies the format of the value and units. `%d` asks only for value in int format. Units are added when `%sb` or `%lb` is used.

Reply:
I see. Thanks.